openslides/agenda/projector.py

```diff
@@ -10,11 +10,41 @@
 
 class ItemListSlide(ProjectorElement):
     """
-    Slide definitions for the agenda.
+    Slide definitions for Item model.
+
+    This is only for list slides.
+
+    Omit 'id' to get the whole agenda tree. Set 'id' to None to get a list
+    slide of all root items. Set 'id' to an integer to get a list slide of
+    the children of the metioned item.
```

Comment:
Ok, I can use `tree` as a property for its own. To get a nested tree on the projector is very useful and only overheaded if you have a huge agenda. Small events normally only have two or three levels.